src/include/ipc/dai-imx.h

```diff
@@ -13,7 +13,7 @@
 
 /* ESAI Configuration Request - SOF_IPC_DAI_ESAI_CONFIG */
 struct sof_ipc_dai_esai_params {
-	struct sof_ipc_hdr hdr;
+	uint32_t reserved0;
```

Comment:
My question right now is why this was introduced in the first place.

Reply:
It was removed a long time ago, then @lgirdwood wanted to have the header for consistency IIRC. I just wanted to highlight it was not used at all and we have 2 words that can be reclaimed.
it's not really useful for this PR/